app/src/main/java/com/dcrandroid/dialog/send/SendDialog.kt

```diff
@@ -127,6 +132,12 @@ class SendDialog(val fragmentActivity: FragmentActivity, dismissListener: Dialog
     override fun onResume() {
         super.onResume()
         destinationAddressCard.addressInputHelper.onResume()
+        if (savedInstanceState != null) {
+            val myAddress = savedInstanceState!!.getString("Address")
+            if(!destinationAddressCard.isSendToAccount){
+                destinationAddressCard.addressInputHelper.editText.setText(myAddress)
```

Comment:
Bundle content should be cleared after restoring but this is getting removed anyways.